tasks/cops_documentation.rake

```diff
@@ -32,7 +32,7 @@ task generate_cops_documentation: :yard do
     content << "--- | ---\n"
     default_status = config.cop_enabled?(cop) ? 'Enabled' : 'Disabled'
     supports_autocorrect = cop.new.support_autocorrect? ? 'Yes' : 'No'
-    content << "#{default_status} | #{supports_autocorrect}"
+    content << "#{default_status} | #{supports_autocorrect}".strip
```

Comment:
I don't see the point for doing `strip` here. Where would an extra space come from?